services/core-web/src/components/admin/MinespaceUserManagement.js

```diff
@@ -68,6 +105,7 @@ export class MinespaceUserManagement extends Component {
         <Divider />
         <br />
         <NewMinespaceUser
+          handleSubmit={this.handleCreateUser}
           refreshData={this.refreshUserData}
           minespaceUserEmailHash={this.props.minespaceUserEmailHash}
```

Comment:
The component doesn't appear to be using this: refreshData
remove if not needed.